Python_Engine/Query/Directory.cs

```diff
@@ -67,6 +67,7 @@ public static string DirectoryEnvironments()
             return dir;
         }
 
+        [PreviousVersion("8.0", "BH.Engine.Python.Query.DirectoryBaseEnvironment()")]
```

Comment:
```suggestion
```
need to remove after beta release - no need to rebase :)

Reply:
I'm good with this change - I didn't realise it would exit 0 if there was already a version installed elsewhere - I was used to seeing error messages for if the same version had been installed previously but somehow failed, so this is a good change.